src/analysis/normed_space/linear_isometry.lean

```diff
@@ -586,6 +607,19 @@ rfl
   ((prod_assoc R E E₂ E₃).symm : E × E₂ × E₃ → (E × E₂) × E₃) = (equiv.prod_assoc E E₂ E₃).symm :=
 rfl
 
+variables {R E E₂ E₃} {R' : Type*} [ring R'] [module R' E] (p q : submodule R' E)
+
+/-- `linear_equiv.of_eq` as a `linear_isometry_equiv`. -/
+@[simps apply symm_apply] def of_eq (hpq : p = q) :
```

Comment:
What lemmas does this generate now then?

Reply:
Okay I should have checked more carefully, because this doesn't give what we want here. The generated lemmas basically says that `linear_isometry_equiv.of_eq p q h x = linear_equiv.of_eq p q h x`, which itself simplifies back to `equiv.set_of_eq`...
So I'll switch back to the manually generated ones